syntax = "proto3";

package x3plus;

// The RosmasterServices definition.
service RosmasterServices {

  rpc SayHello (HelloRequest) returns (HelloReply) {}

}

// The request message containing the user's name.
message HelloRequest {
  string name = 1;
}

// The response message containing the greetings
message HelloReply {
  string message = 1;
}
